free-courses-en.md

```diff
@@ -362,14 +362,14 @@
 * [Codesdope](https://www.codesdope.com/python-introduction)
 * [Google's Python Course](https://developers.google.com/edu/python/)
 * [Introduction to Computer Science and Programming](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-00sc-introduction-to-computer-science-and-programming-spring-2011/) (MIT's opencourseware)
+* [Introduction to Python](https://docs.microsoft.com/en-us/learn/modules/intro-to-python/?WT.mc_id=python-c9-niner)(Microsoft Docs)
```

Comment:
good resource, but I think the querystring is extraeous https://docs.microsoft.com/en-us/learn/modules/intro-to-python/